packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -42,7 +42,12 @@ class ExpansionTile extends StatefulWidget {
     this.trailing,
     this.initiallyExpanded = false,
     this.tilePadding,
+    this.crossAxisAlignment = CrossAxisAlignment.center,
```

Comment:
Please don't provide default values here. In the future, if we support configuring these properties with a theme, we'll need to know if they haven't been set.

Reply:
@HansMuller I have done the appropriate changes and removed the default values from the Constructor.